src/main/kotlin/org/rust/ide/intentions/ExpandModule.kt

```diff
@@ -17,7 +20,7 @@ class ExpandModule : IntentionAction {
     override fun startInWriteAction()   = true
 
     override fun isAvailable(project: Project, editor: Editor?, file: PsiFile?) =
-        file is RustFile && file.name != RustModules.MOD_RS
+        file is RustFile && (!(file.rustMod as RustModItem).ownsDirectory && !(file.name.equals(RustModules.LIB_RS) && file.containingDirectory?.name.equals("src")))
```

Comment:
ok i will make it more null safety, but in `mod.rs` file `owsDirectory` returns true and in `src/lib.rs` this func returns false, it's impossible to handle this two condition use this single function, maybe i should add more check in `owsDirectory`?